DistributionPackages/Neos.NeosIo/Configuration/NodeTypes.VideoEmbed.yaml

```diff
@@ -0,0 +1,21 @@
+'Neos.NeosIo:VideoEmbed':
+  superTypes:
+    'Neos.Neos:Content': true
+  ui:
+    label: Video Embed
+    icon: icon-video
+    group: special
+    inspector:
+      groups:
+        appearance:
+          label: Appearance
+  properties:
+    'mp4':
+      type: 'Neos\Media\Domain\Model\Asset'
+      ui:
+        label: 'MP4 Video'
+        reloadIfChanged: true
+        inspector:
+          group: appearance
+          editiorOptions:
+            accept: mp4
```

Comment:
I would make the property as an array as you perhaps want to add more than one Source:
```
   assets:
      type: array<Neos\Media\Domain\Model\Asset>
      ui:
        label: Videos
        reloadIfChanged: true
        help:
          message: 'The asssets must be `.mp4`, `.webm` or `.ogg` files. Other filetypes will be ignored.'
        inspector:
          group: appearance
          editorOptions:
            accept: 'video/*'
```

Reply:
The editor option `accept` doesn't prevent the user to able to load wrong files in the inspector. So we have to take care of that in Fusion

Reply:
@jonnitto ok, but since mp4 ist the best-supported encoding it should be mandatory, shouldn't it? How about an explicit mp4 property and assets as optional options?

Reply:
I would the mandatory in Fusion. Something like
`<video @if.set={props.sources && String.indexOf(props.sources, 'video/mp4') != -1}`

Reply:
I've updated the code example below